src/common/i18n/translations/en.ts

```diff
@@ -116,7 +116,10 @@ export const en: Translations = {
     appleStore: "App Store",
     androidStore: "Play Store",
   },
-  addonsToggleComponent: { "*chargeable": "Chargeable" },
+  addonsToggleComponent: {
+    "*chargeable": "Chargeable",
+    "*waive charges": "waive charges",
+  },
```

Comment:
Why does `*chargeable` map to `Chargeable`, but `*waive charges` map to `waive charges` and not `Waive charges` 🤔

Reply:
this one is based from the story!